src/graphics/program-lib/chunks/reflectionPrefilteredCube.frag

```diff
@@ -15,50 +15,37 @@ vec3 calcReflection(vec3 tReflDirW, float tGlossiness) {
     // Mips smaller than 4x4 aren't great even for diffuse. Don't forget that we don't have bilinear filtering between different faces.
 
     float bias = saturate(1.0 - tGlossiness) * 5.0; // multiply by max mip level
-    int index1 = int(bias);
-    int index2 = int(min(bias + 1.0, 7.0));
-
-    vec3 fixedReflDir = fixSeams(cubeMapProject(tReflDirW), bias);
-    fixedReflDir.x *= -1.0;
-
-    vec4 cubes[6];
-    cubes[0] = textureCube(texture_prefilteredCubeMap128, fixedReflDir);
-    cubes[1] = textureCube(texture_prefilteredCubeMap64, fixedReflDir);
-    cubes[2] = textureCube(texture_prefilteredCubeMap32, fixedReflDir);
-    cubes[3] = textureCube(texture_prefilteredCubeMap16, fixedReflDir);
-    cubes[4] = textureCube(texture_prefilteredCubeMap8, fixedReflDir);
-    cubes[5] = textureCube(texture_prefilteredCubeMap4, fixedReflDir);
-
-    // Also we don't have dynamic indexing in PS, so...
-    vec4 cube[2];
-    for(int i = 0; i < 6; i++) {
-        if (i == index1) {
-            cube[0] = cubes[i];
-        }
-        if (i == index2) {
-            cube[1] = cubes[i];
-        }
+    vec3 refl = cubeMapProject(tReflDirW);
```

Comment:
So to be clear, if anyone has a project where they took `reflectionPrefilteredCube.frag` and modified it, would their project need updating now? Or would it still work>